content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md

```diff
@@ -0,0 +1,126 @@
+<div class="box-info">Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.</div>
+
+The required ISO file can be found at on the "Get Haiku" page of this website which is located [here](/get-haiku). VMware Player is available for free on their website which can be found [here](https://www.vmware.com/products/player/). There is also VMware Workstation, which is similar but has more features and is commercial. This means you have to pay for this version, but it does offer a 30 days trial. However you could just stick to the free one _(VMware Player)_ as you most likely will not need the extra features. You can both read about the commercial version's extra features and download it from its website [here](https://www.vmware.com/products/workstation/overview.html)
+
+##### Go to section:
+
+*   [Running Haiku from a VM image.](#part_vmimage)
+*   [Installing and running Haiku from an ISO image.](#part_iso)
+*   [Additional Steps.](#part_additional)
+*   [Troubleshooting.](#part_trouble)
+
+### Running Haiku from a VM image.
+
+##### Step 1. Downloading Haiku.
+
+Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](/files/image1.jpeg)

```

Comment:
You'll need to add the images to the repository in the proper location and adjust all the paths appropriately.